### PR TITLE
Remove duplicates

### PR DESCRIPTION
**Pull request by bernardoamc**
@erichkist @laurocaetano can you guys review this post? :)

### CODE REVIEW

File: _posts/2015-08-11-postgres-remove-duplicates.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  3 |       2 | dba
+  4 |       2 | hjk
+  5 |       2 | dba
+  6 |       3 | fkl
+(6 rows)
+{% endhighlight %}
+
+We want to remove rows that have the same `item_id` and `code`, what are our
+options? Actually, a lot! (If your table doesn't  have a unique identifier it's
+useful to read [this](#ctid) section first.)
```
Comment (laurocaetano) [inline _posts/2015-08-11-postgres-remove-duplicates.md]: What is #ctid?

Comment (laurocaetano) [inline _posts/2015-08-11-postgres-remove-duplicates.md]: hmm, now I got it hahaha

File: _posts/2015-08-11-postgres-remove-duplicates.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  2 |       1 | abc
+  3 |       2 | dba
+  4 |       2 | hjk
+  5 |       2 | dba
+  6 |       3 | fkl
+(6 rows)
+{% endhighlight %}
+
+We want to remove rows that have the same `item_id` and `code`, what are our
+options? Actually, a lot! (If your table doesn't  have a unique identifier it's
```
Comment (erichkist) [inline _posts/2015-08-11-postgres-remove-duplicates.md]: There is a extra blank space in the phrase between `doesn't` and `have`: `If your table doesn't  have` .

File: _posts/2015-08-11-postgres-remove-duplicates.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+possibility that we should take into account.
+
+We can also use a `CREATE TEMPORARY TABLE` if our table should be dropped at the
+end of the session.
+
+### <a name="ctid"></a>But my rows don't have a unique identifier!
+
+In this case you can rename the `id` to `ctid` in our queries.
+
+`ctid` is a [system column](http://www.postgresql.org/docs/9.4/static/ddl-system-columns.html)
```
Comment (laurocaetano) [inline _posts/2015-08-11-postgres-remove-duplicates.md]: Do you have any example for this usage?

File: _posts/2015-08-11-postgres-remove-duplicates.md (hunk 1 above)
Comment (erichkist) [inline _posts/2015-08-11-postgres-remove-duplicates.md]: lol.. I think the `anchor` confused me.. Maybe you can start this information here as a section necessary to the other.

Regarding the `anchor` strategy, I think the markdown has something for this (you don't need to create the `a` tag)

File: _posts/2015-08-11-postgres-remove-duplicates.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+  FROM (
+   SELECT row_number() OVER (PARTITION BY item_id, code ORDER BY id),
+          id
+   FROM orders
+  ) AS sorted
+  WHERE sorted.row_number > 1
+);
+{% endhighlight %}
+
+Here we partition our records using the specified criteria to find duplicates
```
Comment (laurocaetano) [inline _posts/2015-08-11-postgres-remove-duplicates.md]: Here the records were partitioned ... ?

File: _posts/2015-08-11-postgres-remove-duplicates.md (hunk 1 above)
Comment (bernardoamc) [inline _posts/2015-08-11-postgres-remove-duplicates.md]: Maybe adding the section here and telling the user to skip it if they have a unique identifier?

Didn't find anywhere how to create anchors in markdown. :(

File: _posts/2015-08-11-postgres-remove-duplicates.md (hunk 3 above)
Comment (bernardoamc) [inline _posts/2015-08-11-postgres-remove-duplicates.md]: Just replacing `id` by `ctid` in the queries above is an example, but I don't know if the postgres docs have an example like this.

File: _posts/2015-08-11-postgres-remove-duplicates.md (hunk 4 above)
Comment (bernardoamc) [inline _posts/2015-08-11-postgres-remove-duplicates.md]: :+1:

File: _posts/2015-08-11-postgres-remove-duplicates.md (hunk 2 above)
Comment (bernardoamc) [inline _posts/2015-08-11-postgres-remove-duplicates.md]: :+1:

**Pull request merged**